imu_filter_madgwick/CMakeLists.txt

```diff
@@ -1,75 +1,66 @@
-cmake_minimum_required(VERSION 3.5.1)
-cmake_policy(SET CMP0048 NEW)
```

Comment:
I believe we need to keep the `cmake_policy(SET CMP0048 NEW)` line, otherwise the build farm will complain about a CMake warning.

Reply:
Done